### PR TITLE
fixed socket error on sendato

### PR DESCRIPTION
**Pull request by navjotcis**
here when the internet connection is disconnected then the socket connection starts giving an exception so in the previous code socket was closed when socket.error produces but it creates an issue because when the user again connects with the internet then it is not connecting with sockets so it causes and issues so I have commented socket close line then it starts working fine.

### CODE REVIEW

File: src/network/udp.py [hunk 1]
```diff
@@ -145,8 +145,9 @@ def handle_write(self):
                 self.write_buf, ('<broadcast>', self.port))
         except socket.error as e:
             logger.error("socket error on sendto: %s", e)
-            if e.errno == 101:
-                self.announcing = False
-                self.socket.close()
             retval = 0
```
Comment (PeterSurda) [inline src/network/udp.py:148]: remove line 148

File: src/network/udp.py [hunk 2]
```diff
@@ -145,8 +145,9 @@ def handle_write(self):
...
+            retval = len(self.write_buf)
+            # if e.errno == 101:
```
Comment (PeterSurda) [inline src/network/udp.py:150]: remove lines 150 through 152

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/network/udp.py (hunk 1 above)
Comment (PeterSurda) [reply src/network/udp.py:148]: Still not done (line 148 in new, 151 in old)

**Review by PeterSurda — CHANGES_REQUESTED** [review]

**Comment by navjotcis** [comment]
I have tried to reproduce the issue but I did not able to reproduce it.

File: src/network/udp.py (hunk 1 above)
Comment (navjotcis) [reply src/network/udp.py]: removed line 148

**Review by PeterSurda — APPROVED** [review]

**Review by PeterSurda — APPROVED** [review]

**Comment by PeterSurda** [comment]
I refactored the use of `state.appdata` and removed the environment variable from `.travis.yml`. It now passes buildbot's bionic and xenial, as well as xenian on travis-ci.org. Can you try rebasing your changes on top of it again?

**Pull request merged**